_topics/block-explorers.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Block explorers
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Privacy Problems
+  - Developer Tools
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+excerpt: >
+  **Block explorers** are tools that maintain an index of transactions
```

Comment:
I tend to think of Electrum Server (and it's clones, like electrs) as block explorers also since they build the same tables as website block explorers.

Reply:
Fair point. Resolving.